### ISSUE TITLE
False-positive W0143:comparison-with-callable with PySide6 and `__feature__`

### ISSUE THREAD

**Issue by bersbersbers** [PyCQA/pylint#7519]
### Bug description

from PySide6.QtWidgets import QApplication, QSlider

from __feature__ import snake_case, true_property  # isort: skip

QApplication()
slider = QSlider()
print(slider.maximum)
print(slider.maximum == 99)

### Command used

```shell
pylint bug.py
```

### Pylint output

```shell
fastview\bug.py:1:0: C0114: Missing module docstring (missing-module-docstring)
fastview\bug.py:3:0: E0401: Unable to import '__feature__' (import-error)
fastview\bug.py:7:6: W0143: Comparing against a callable, did you omit the parenthesis? (comparison-with-callable)
fastview\bug.py:3:0: W0611: Unused snake_case imported from __feature__ (unused-import)
fastview\bug.py:3:0: W0611: Unused true_property imported from __feature__ (unused-import)

-----------------------------------
Your code has been rated at 0.00/10
```

### Expected behavior

No warning about the callable.

### Pylint version

```shell
pylint 2.15.3
astroid 2.12.10
Python 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)]
```

### OS / Environment

Windows 10

### Additional dependencies

PySide6            6.3.2
PySide6-Addons     6.3.2
PySide6-Essentials 6.3.2
